pkgs/development/python-modules/art/default.nix

```
@@ -6,14 +6,14 @@
 
 buildPythonPackage rec {
   pname = "art";
-  version = "6.2";
+  version = "6.3";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and specify `build-system`